### ISSUE TITLE
User Guide structure

### ISSUE THREAD

**Issue by Mehvin** [Mehvin/pe#6]
Though the user guide has alot of information which was well done, it is not very easy to follow for a new user.

Instead of throwing all the infomation to the user in the beginning, it can instead be left to the end or completely omitted.

Reccomendation: the more important parts should maybe be at the start such as the `commands` section as this shows a rough overview of what the products does (even though the user might not understand it initially). 

Though the earlier parts makes sense to be at the beginning, it can be placed at the end as well. For example, the `command format` and the `trying your first command` section can possibly in an appendix, for users to look into if they needed more guidance.

Tldr, though I feel that the UG was well done with lots of information, a UG should be kept brief and not too long.

**Comment by nus-pe-script** [comment]
# Team's Response
Thanks for the detailed suggestion!
# Items for the Tester to Verify
## :question: Issue response

Team chose [`response.NotInScope`]

- [ ] I disagree

**Reason for disagreement:** [replace this with your explanation]

<catcher-end-of-segment><hr>
